README.rst

```diff
@@ -0,0 +1,59 @@
+# Princeton Geniza Project Research Partnership
+
+
+Python/Django web application for a new version of the `Princeton Geniza Project
+<https://cdh.princeton.edu/projects/princeton-geniza-project/>`_.
+
+Python 3.8 / Django 3.1 / Postgresql
+
+
+.. image:: https://github.com/Princeton-CDH/geniza/workflows/unit_tests/badge.svg
+    :target: https://github.com/Princeton-CDH/geniza/actions?query=workflow%3Aunit_tests
+    :alt: Unit Test status
+
+.. image:: https://codecov.io/gh/Princeton-CDH/geniza/branch/main/graph/badge.svg
+   :target: https://codecov.io/gh/Princeton-CDH/geniza
+   :alt: Code coverage
+
+.. image:: https://requires.io/github/Princeton-CDH/geniza/requirements.svg?branch=main
+     :target: https://requires.io/github/Princeton-CDH/geniza/requirements/?branch=main
+     :alt: Requirements Status
+
+.. image:: https://github.com/Princeton-CDH/geniza/workflows/dbdocs/badge.svg
+    :target: https://dbdocs.io/princetoncdh/geniza
+    :alt: dbdocs build
+
+
+Development instructions
+------------------------
+
+Initial setup and installation:
+
+- Recommended: create and activate a python 3.8 virtualenv
+
+- Install required python dependencies::
+
+    pip install -r requirements/dev.txt
+
+- Copy sample local settings and configure for your environment::
+
+	cp geniza/settings/local_settings.py.sample geniza/settings/local_settings.py
+
+Remember to add a ``SECRET_KEY`` setting!
+
+- Create a new database and update your database settings accordingly
+
+- Run database migrations
+
+    python manage.py migrate
+
+
+Unit Tests
+----------
+
+Python unit tests are written with `py.test <http://doc.pytest.org/>`_
+and should be run with `pytest`.
+
+License
+-------
+This project is licensed under the `Apache 2.0 License <https://github.com/Princeton-CDH/mep-django/blob/main/LICENSE>`_.
```

Comment:
I've had issues with rst rendering on GitHub. Has it been just me? What are the unique benefits to rst?

Reply:
One benefit I know of for us is that it allows sphinx to pick up the README and use it as the frontpage of the generated code documentation - here's [an example for PPA](https://princeton-cdh.github.io/ppa-django/). I think sphinx is still the de facto standard for autogenerated codedocs for python projects, which is why we use it (although I've been playing with [`pdoc3`](https://pdoc3.github.io/pdoc/) for DIRECT and like it so far).

Reply:
yes, exactly — I agree that GitHub's support for rst is fairly limited, but having the readme in rst means we can easily include it in our sphinx docs when we set them up. 

@thatbudakguy I saw you mentioned using pdoc3 for direct and wondered about it. I'd be open to trying that for geniza if you think it's a better choice and does everything we need it to!